.github/workflows/build_and_run_tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       uses: robinraju/release-downloader@v1.6
       with:
         repository: "mesonbuild/meson"
-        tag: "0.53.2"
+        tag: "1.7.0"
```

Comment:
I'm not sure why we do this?
 Usually we use the lowest version we promise to support (promise in `meson.build`), and that did not change.

On the contrary we say `meson_version : '>=0.49'`.

Whatever, I usually use `HEAD` of Meson locally, and if we use the lowest allowed version in the CI we get a nice test range.

Additionally I think we should use the Meson version that is system-package-available on the test-OS (if it suffices).

In this way we would use 0.53.2 on 20.04 and 1.3.2 on 24.04.

Reply:
Would it make sense to use 0.54.1 here?